SR2024/2024-04-04-kit-recovery.md

```diff
@@ -0,0 +1,31 @@
+---
+to: Teams with outstanding kit
+subject: Outstanding kit
+attachment: https://drive.google.com/file/d/1JQG_IKOtnvEUPSob1XWTma55FHRK_r4o/view
+---
+
+Hello!
+
+According to our records, you still have a kit from this years competition. As the competition has ended, we would like this back!
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). You can find a list of the parts to return attached to this email or [on our website](https://studentrobotics.org/docs/kit/).
```

Comment:
The link texts here aren't super useful as descriptions of what they link to. Best practise for links is that the link text should be as descriptive as possible, so perhaps we could improve this? This is especially important for any users of assistive technologies which may use the link text to indicate which one they want to visit.

In this case there are two links which go to different places yet have the same link text!

```suggestion
Please securely pack the kit and ship it to our [postal address](https://studentrobotics.org/contact/). You can find a [list of the parts](https://studentrobotics.org/docs/kit/) to return on our website or as a checklist attached to this email.
```